index.html

```diff
@@ -2,94 +2,111 @@
+            <div class="col-lg-4">
+                <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
+                    <div class="card-header">restaurant name</div>
+                    <div class="card-body">
+                        <h5 class="card-title">restaurant rating</h5>
+                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
+                            the card's content.</p>
+                    </div>
                 </div>
             </div>
-        </div>
 
-<div class="col-lg-4">
-    <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
-        <div class="card-header">restaurant name</div>
-            <div class="card-body">
-                <h5 class="card-title">restaurant rating</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+            <div class="col-lg-4">
+                <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
+                    <div class="card-header">restaurant name</div>
+                    <div class="card-body">
+                        <h5 class="card-title">restaurant rating</h5>
+                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
+                            the card's content.</p>
+                    </div>
                 </div>
             </div>
-        </div>
 
-<div class="col-lg-4">
-    <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
-        <div class="card-header">restaurant name</div>
-            <div class="card-body">
-                <h5 class="card-title">restaurant rating</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+            <div class="col-lg-4">
+                <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
+                    <div class="card-header">restaurant name</div>
+                    <div class="card-body">
+                        <h5 class="card-title">restaurant rating</h5>
+                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
+                            the card's content.</p>
+                    </div>
                 </div>
             </div>
         </div>
-    </div>
 
-<h4 class="option">recipes options:</h4>
+        <h4 class="option">recipes options:</h4>
 
-<div class="row">
-        <div class="col-lg-4">
-            <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
-                <div class="card-header">Recipe</div>
+        <div class="row">
+            <div class="col-lg-4">
+                <div class="card card-one text-white bg-secondary mb-3" style="max-width: 20rem;">
```

Comment:
Please use an ID here instead of altering the class name.  If you wanted to use an ID of card one that totally works!